test/__init__.py

```diff
@@ -84,25 +90,25 @@ def has_alpn(ctx_cls=None):
 RESOLVES_LOCALHOST_FQDN = _can_resolve("localhost.")
 
 
-def clear_warnings(cls=HTTPWarning):
+def clear_warnings(cls: type = HTTPWarning) -> None:
     new_filters = []
-    for f in warnings.filters:
+    for f in warnings.filters:  # type: ignore
```

Comment:
If you feel like it, can send a patch to typeshed adding `filters` to `warnings.pyi` as an undocumented attribute.